assets/js/components/HostDetails/HostChecksSelection.jsx

```diff
@@ -29,16 +31,12 @@ function HostChecksSelection({
         catalogError={catalogError}
         loading={catalogLoading}
         selected={selectedChecks}
-        onSave={(_selectedChecks, _hostID) => {
-          // TODO: dispatch check selection for a host
-        }}
+        onSave={(selection, targetID) => onSaveSelection(selection, targetID)}
```

Comment:
I would personally continue using the `selectedChecks` term.
Adding `selection` adds yet another term for the same concept.
Or use `selection` generally

Reply:
again name clashing here.
`selection` here is the possibly different selection emitted when a user clicks on the save selection button, not the `selectedChecks` used to initially feed the component.

Maybe rename it to `newSelectedChecks`/`newChecksSelection`?

Reply:
renamed the function argument to `newSelectedChecks`